docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,98 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+
+Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+include::{includes}/extension-status.adoc[]
+
+== Introduction
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running an update command.
+The update command uses OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation.
+
+// Give the link to the OpenRewrite recipe?
+// How does the user know what the update command does or does not cover?
+// In the migration guide, can we identify which items are/are not covered by the OpenRewrite recipe?
+// Do we want to invite feedback? For example: If it does not work for you, we could have missed something, or you are using extensions not yet supporting Quarkus 3.
+// https://github.com/quarkusio/quarkus/issues[Let us know] in either case so we can improve the migration script together.
+
+
+[IMPORTANT]
+====
+After updating, evaluate whether your project requires additional follow-up steps.
+Verify that the application from your updated project works as expected before releasing it to production.
+====
+
+== Prerequisites
+
+:prerequisites-time: 30 minutes
+include::{includes}/prerequisites.adoc[]
+
+[IMPORTANT]
+====
+If your project uses Hibernate ORM or Hibernate Reactive, see the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] quick reference.
```

Comment:
@yrodiere 
What's the best placement for this note? 
Does the update command take care of any of the changes described in this quick reference?

Reply:
>  What's the best placement for this note?

I'd merge it into the IMPORTANT block above that starts with "After updating, evaluate whether your project requires additional follow-up steps"?

E.g. :

```
[IMPORTANT]
====
After updating, evaluate whether your project requires additional follow-up steps.

Verify that the application from your updated project works as expected before releasing it to production.

In particular, if your project uses Hibernate ORM or Hibernate Reactive, you will want to read through the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] quick reference.
====

>  Does the update command take care of any of the changes described in this quick reference?

Some, yes; most simply cannot be automated. 

The package changes (`javax.persistence` => `jakarta.persistence`) is definitely handled, and I think @gsmet was considering enabling the [Hibernate ORM 5.6 compatibility switch](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration#best-effort-hibernate-orm-56-compatibility-switch), which should address most changes in the "Database schema" section, but I'm not sure he got to doing it.

Regardless... Surely Hibernate ORM is not the only library with changes that are not handled by `quarkus upgrade`. Shouldn't this problem be approached more globally for the whole migration guide? Like linking to the migration guide from here, and somehow documenting in the migration guide whether each section of the migration guide is handled by `quarkus upgrade` or not.

Reply:
Updated. Thanks.